packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -583,6 +583,21 @@ class DedicatedCloudService {
     });
     return filterRequest;
   }
+
+  postGrantActiveDirectory(serviceName, activeDirectoryId, params, type) {
```

Comment:
nitpick 
```suggestion
  grantActiveDirectory(serviceName, activeDirectoryId, params, type) {
```
grant is already an action so no need to prefix by antoher action

Reply:
It's applied